src/masquerade/main.py

```diff
@@ -292,6 +296,42 @@ def geocode_addresses():
     }
 
 
+@app.route(f"{GEOCODE_SERVER_ROUTE}/reverseGeocode", methods=["GET", "POST"])
+@as_json_p
+def reverse_geocode():
+    """reverse geocode a point"""
+
+    request_wkid, out_spatial_reference = get_out_spatial_reference(request)
+
+    try:
+        location = json.loads(get_request_param(request, "location"))
+    except json.JSONDecodeError:
+        return {"error": "location param is not valid JSON"}, 400
+
+    if location["spatialReference"]["wkid"] != out_spatial_reference:
+        from_crs = CRS.from_epsg(location["spatialReference"]["wkid"])
+        to_crs = CRS.from_epsg(out_spatial_reference)
+        transformer = Transformer.from_crs(from_crs, to_crs, always_xy=True)
+        x, y = transformer.transform(location["x"], location["y"])
+    else:
+        x, y = location["x"], location["y"]
+
+    result = web_api.reverse_geocode(x, y, out_spatial_reference, location["x"], location["y"])
```

Comment:
**question**: Since this can throw, do you want it not to be caught?

Reply:
If the handler throws, it returns a 500. I think that I'm OK with that but totally open to suggestions if you have any.